wiki/Gameplay/Input_device/Drum_controller/en.md

```diff
@@ -0,0 +1,8 @@
+# Drum Controller
+
+**Drum controllers** are input devices designed to work with taiko drum games. 
```

Comment:
```suggestion
**Drum controllers** are input devices designed to work with Taiko-like drum games. They most often have four areas of input: right drum center, left drum center, right drum rim, and left drum rim, with each input (key) being activated via a drum stick that is smacked down on the corresponding area of the drum.
```
**Some images of popular taiko drum controllers should be placed here as well.** "Taiko" should prob have a link somewhere. Not sure if we want to link to an article within osu! wiki or to Wikipedia, but it should be linked.